app/assets/stylesheets/views/_covid.scss

```diff
@@ -51,6 +51,18 @@
   }
 }
 
+.covid__page-header-announcement {
+  margin-bottom: govuk-spacing(3);
+
+  @include govuk-media-query($until: desktop) {
+    margin-bottom: govuk-spacing(7);
```

Comment:
It makes no real difference, but I feel instinctively that we should write our code mobile first, so this would be 

```
.covid__page-header-announcement {
  margin-bottom: govuk-spacing(7);

  @include govuk-media-query($from: desktop) {
    margin-bottom: govuk-spacing(3);
  }
```

Reply:
> Seems okay, I'd like to check the review app but it doesn't seem to have deployed yet?

Github was down for a bit yesterday which broke a few things, I'll get it to trigger again.